guides/example-app-walkthrough.md

```diff
@@ -0,0 +1,217 @@
+- [`epic`(add link to epic below)](): a function of shape `(action$, store, { feathers }) => nextAction$`, combined from each topic using [`combineEpics`](https://redux-observable.js.org/docs/api/combineEpics.html)
+- [`middlewares`](http://redux.js.org/docs/Glossary.html#middleware): an array of functions of shape `store => next => action`
+- [`enhancers`](http://redux.js.org/docs/Glossary.html#store-enhancer): an array of functions of shape `createStore => createEnhancedStore
+
+```js
+// store.js
+import updater from './updater'
+import epic from './epic'
+
+const middlewares = []
+const enhancers = []
+
+export default {
+  updater,
+  epic,
+  middlewares,
+  enhancers
+}
+```
+
+
+#### updater.js
+
+> An Updater is a type of function that takes an action and returns another function that takes the state and produces a result. Yes, they are just curried reducers with reversed argument order.
+
+If you're used to the term `reducer`, you can see in our `updater.js` file we have a function that converts reducers to updaters
+
+```js
+function reducerToUpdater (reducer) {
+  return action => state => reducer(state, action)
+}
+``` 
+
+
+This file combines all of our apps `updaters` using [`redux-fp.concat`](https://github.com/rvikmanis/redux-fp/blob/master/docs/API.md#concat). Think of this as your 'root updater'. For any updaters to interact with the state, they must be in this file.
+
+#### epic.js
+
+This file exports a function of shape `(action$, store, { feathers }) => nextAction$`, combined from each topic using [`combineEpics`](https://redux-observable.js.org/docs/api/combineEpics.html)
+
+> An Epic is a function which takes a stream of actions and returns a stream of actions. Actions in, actions out.
+
+Need this looked at, still confusing to me after looking into `dogstack-agents`
+The main way dogstack uses epics is with Feathers. We can subscribe to events, the epic will define what events we want to subscribe to, and what happens when the event is triggered.
+
+Luckily [feathers-action](https://www.npmjs.com/package/feathers-action) does most of the heavy lifting for us and this will be the main way to create epics, but it helps to know the basics of [Observables and RxJs](http://reactivex.io/rxjs/manual/overview.html)
+
+#### client.js - required by dogstack
+
+This file is used for our [`feathers` client](https://docs.feathersjs.com/api/client.html) configuration(yes! feathers is also a client) so that know how to communicate with our feathers server and connect things like authentication.
+
+This file is used to define what services we want the feathers client to use.
+
+`services`: export an array of functions that will be run with [`client.configure(plugin)`](https://docs.feathersjs.com/api/application.html#configurecallback)
+
+#### root.js
+
+To ask Mikey - what's the purpose of styleNode in dogstack (in root.js)? I see where we use appNode, but don't see styleNode being used at all?
```

Comment:
since https://github.com/root-systems/dogstack/commit/0197d3ec59cd4299839f51c426988de71f6ccc39, `styleNode` is no longer used